### ISSUE TITLE
Is there a little differnece between the code and the paper?

### ISSUE THREAD

**Issue by GuideWsp** [ykshi/DDet#1]
I found the kernel size in paper is 3-5-7, while the kernel size of the code is 5-7-9.

**Comment by ykshi** [comment]
> I found the kernel size in paper is 3-5-7, while the kernel size of the code is 5-7-9.

Yes, the kernel size is 5-7-9, I wrote wrong number by mistake. The corresponding content will be revised in next version. Thank you very much~
